### ISSUE TITLE
Request for Model Weights

### ISSUE THREAD

**Issue by GMrrc** [NcolasDran/sketch-colorization-based-on-reference-structure#2]
Hello, I tried to train a model using the given dataset, but after 110 epochs, the results are still unsatisfactory, with the discriminator's loss not going below 6. Is there a place where I can find your model weights?

**Comment by NcolasDran** [comment]
Hi, I apologized for the very late response, unfortunately I don't have the model weights.

But maybe your scenario is better than mine, because in the original paper the Adversarial Loss remained constant with a value of 3, but in my case it dropped to almost 0 after epoch 25, and my Reconstruction Loss was approximately 8.25 at epoch 100. And I have to say that my results are not satisfactory either in my opinion, so it may be necessary to modify the architecture or improve the training dataset.
